terraform/staging_pipeline.tf

```diff
@@ -51,6 +51,13 @@ resource "heroku_formation" "api_staging_checksum_worker" {
   quantity = 1
 }
 
+resource "heroku_formation" "api_staging_checksum_worker" {
```

Comment:
Why does this have the same name as the above checksum worker? Shouldn't this be named `api_staging_manifest_worker` or something?

Reply:
good catch, that is correct